pep-0604.rst

```diff
@@ -163,16 +209,26 @@ if we accept to not be compatible with the dynamic evaluation of annotations (``
 PROS:
 
 - If they were permitted, then instance checking could use an extremely clean-looking notation
-- The implementation can use the tuple present in ``Union`` parameter, without create a new instance
+- The implementation can use the tuple present in ``Union`` parameter, without creating a new instance
```

Comment:
In #1540, @methane proposes to strike this sentence, as it is not a real example. (Probably `int|float|str` is actually slower than `(int, float, str)`, which maybe should be listed under CONS.)